<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UCreateOrder</class>
 <widget class="QWidget" name="UCreateOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>寄件申请</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>511</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_recievename"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_sendname"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>寄件人电话</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>寄件人</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_sendphone"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>收件人</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>收件人电话</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_recievephone"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>寄件人地址</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit_sendAddr"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>收件人地址</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QTextEdit" name="textEdit_recieveAddr"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>处理站点</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>物品重量</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_weight"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>预计运费</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_payment"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_station">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站0号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站1号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站2号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站3号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站4号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站5号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站6号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站7号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站8号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菜鸟驿站9号</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_commit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>快递单号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Oid"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
